homeassistant/components/smartthings/strings.json

```diff
@@ -477,6 +477,10 @@
     "deprecated_binary_fridge_door": {
       "title": "Deprecated refrigerator door binary sensor detected in some automations or scripts",
       "description": "The refrigerator door binary sensor `{entity}` is deprecated and is used in the following automations or scripts:\n{items}\n\nSeparate entities for cooler and freezer door are available and should be used going forward. Please use them in the above automations or scripts to fix this issue."
+    },
+    "deprecated_switch_appliance": {
+      "title": "Deprecated switch detected in some automations or scripts",
+      "description": "The switch `{entity}` is deprecated and is used in the following automations or scripts:\n{items}\n\nThe actions of the switch did not work, and it has been replaced with a binary sensor instead. Please use them in the above automations or scripts to fix this issue."
```

Comment:
I'm suggesting adding the items as last as if a user is using it in multiple automations/script it can look strange